Index: Assets/Scripts/SettingsController.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public static class SettingsController\n{\n    public static string DeviceController { get; set; } = \"steeringWheel\"; //for using ulta mooie stuurtje oehleh\n    // public static string DeviceController { get; set; } = \"gamepadController\"; // Controller (ps4)\n    // public static string DeviceController { get; set; } = \"keyboardController\";  // Keyboard\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/Scripts/SettingsController.cs b/Assets/Scripts/SettingsController.cs
--- a/Assets/Scripts/SettingsController.cs	(revision 949b3da38bcae2a0fdb8346f31f565545a0e5dee)
+++ b/Assets/Scripts/SettingsController.cs	(date 1737366194634)
@@ -1,6 +1,6 @@
 public static class SettingsController
 {
-    public static string DeviceController { get; set; } = "steeringWheel"; //for using ulta mooie stuurtje oehleh
+    // public static string DeviceController { get; set; } = "steeringWheel"; //for using ulta mooie stuurtje oehleh
     // public static string DeviceController { get; set; } = "gamepadController"; // Controller (ps4)
-    // public static string DeviceController { get; set; } = "keyboardController";  // Keyboard
+    public static string DeviceController { get; set; } = "keyboardController";  // Keyboard
 }
Index: Assets/TestScenes/SampleScene.unity
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assets/TestScenes/SampleScene.unity b/Assets/TestScenes/SampleScene.unity
--- a/Assets/TestScenes/SampleScene.unity	(revision 949b3da38bcae2a0fdb8346f31f565545a0e5dee)
+++ b/Assets/TestScenes/SampleScene.unity	(date 1737366167271)
@@ -251,6 +251,12 @@
     m_RemovedGameObjects: []
     m_AddedGameObjects: []
     m_AddedComponents:
+    - targetCorrespondingSourceObject: {fileID: 5551360141245956375, guid: 47b41a8f5cd80894aa9bd0933d2b52fe, type: 3}
+      insertIndex: -1
+      addedObject: {fileID: 1181191710}
+    - targetCorrespondingSourceObject: {fileID: 5551360141245956375, guid: 47b41a8f5cd80894aa9bd0933d2b52fe, type: 3}
+      insertIndex: -1
+      addedObject: {fileID: 1181191711}
     - targetCorrespondingSourceObject: {fileID: 5267775917890738222, guid: 47b41a8f5cd80894aa9bd0933d2b52fe, type: 3}
       insertIndex: -1
       addedObject: {fileID: 2089662314}
@@ -273,12 +279,17 @@
       insertIndex: -1
       addedObject: {fileID: 1812642885}
   m_SourcePrefab: {fileID: 100100000, guid: 47b41a8f5cd80894aa9bd0933d2b52fe, type: 3}
+--- !u!1 &214930150 stripped
+GameObject:
+  m_CorrespondingSourceObject: {fileID: 5551360141245956375, guid: 47b41a8f5cd80894aa9bd0933d2b52fe, type: 3}
+  m_PrefabInstance: {fileID: 214930149}
+  m_PrefabAsset: {fileID: 0}
 --- !u!114 &214930151 stripped
 MonoBehaviour:
   m_CorrespondingSourceObject: {fileID: 6864331406162114436, guid: 47b41a8f5cd80894aa9bd0933d2b52fe, type: 3}
   m_PrefabInstance: {fileID: 214930149}
   m_PrefabAsset: {fileID: 0}
-  m_GameObject: {fileID: 0}
+  m_GameObject: {fileID: 214930150}
   m_Enabled: 1
   m_EditorHideFlags: 0
   m_Script: {fileID: 11500000, guid: 78942c6e946a7324ca7f2609e5671f81, type: 3}
@@ -141389,6 +141400,51 @@
   m_CorrespondingSourceObject: {fileID: 5062525429664586669, guid: 47b41a8f5cd80894aa9bd0933d2b52fe, type: 3}
   m_PrefabInstance: {fileID: 214930149}
   m_PrefabAsset: {fileID: 0}
+--- !u!114 &1181191710
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 214930150}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d5a57f767e5e46a458fc5d3c628d0cbb, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  GlobalObjectIdHash: 3858207748
+  InScenePlacedSourceGlobalObjectIdHash: 2294311777
+  DeferredDespawnTick: 0
+  Ownership: 1
+  AlwaysReplicateAsRoot: 0
+  SynchronizeTransform: 1
+  ActiveSceneSynchronization: 0
+  SceneMigrationSynchronization: 1
+  SpawnWithObservers: 1
+  DontDestroyWithOwner: 0
+  AutoObjectParentSync: 1
+  SyncOwnerTransformWhenParented: 1
+  AllowOwnerToParent: 0
+--- !u!114 &1181191711
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 214930150}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 6f573b2417ce4e52853c2a28123e074f, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  ShowTopMostFoldoutHeaderGroup: 1
+  pushPower: 20
+  rb: {fileID: 0}
+  position: 0
+  totalDriveTime: 0
+  shortName: 
+  time: 0
+  tire: 
 --- !u!1 &1274105397
 GameObject:
   m_ObjectHideFlags: 0
@@ -142212,6 +142268,18 @@
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 2147483647
   m_IsActive: 1
+--- !u!114 &703439785121964108
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 6919946283301097841}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: a0ed553bf2b54719b9363b6f62fa7e91, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
 --- !u!4 &724472978026749492
 Transform:
   m_ObjectHideFlags: 0
@@ -143356,6 +143424,27 @@
   m_Children: []
   m_Father: {fileID: 8288199812752346961}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+--- !u!65 &1958306796285454634
+BoxCollider:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 7174552832594421700}
+  m_Material: {fileID: 0}
+  m_IncludeLayers:
+    serializedVersion: 2
+    m_Bits: 0
+  m_ExcludeLayers:
+    serializedVersion: 2
+    m_Bits: 0
+  m_LayerOverridePriority: 0
+  m_IsTrigger: 1
+  m_ProvidesContacts: 0
+  m_Enabled: 1
+  serializedVersion: 3
+  m_Size: {x: 1, y: 1, z: 1}
+  m_Center: {x: 0, y: 0, z: 0}
 --- !u!1 &1969659787326816148
 GameObject:
   m_ObjectHideFlags: 0
@@ -145009,6 +145098,22 @@
   m_Children: []
   m_Father: {fileID: 8288199812752346961}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+--- !u!114 &3407610695652645223
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 6919946283301097841}
+  m_Enabled: 0
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: ae24165181134e0184c08d18ad2ef5e9, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  ShowTopMostFoldoutHeaderGroup: 1
+  test: 1
+  leaderBoard: {fileID: 0}
+  playerData: {fileID: 0}
 --- !u!1001 &3469063806810633287
 PrefabInstance:
   m_ObjectHideFlags: 0
@@ -145173,6 +145278,22 @@
   m_Children: []
   m_Father: {fileID: 8288199812752346961}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+--- !u!114 &3500047205986739812
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 9095071193264868738}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: e4cec0a24e334589955225595d5effbb, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  ShowTopMostFoldoutHeaderGroup: 1
+  isFinish: 1
+  leaderBoard: {fileID: 0}
+  sectorId: 1
 --- !u!4 &3505358639611146499
 Transform:
   m_ObjectHideFlags: 0
@@ -145897,6 +146018,31 @@
   m_Children: []
   m_Father: {fileID: 8288199812752346961}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+--- !u!114 &4138672750598196639
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 9095071193264868738}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d5a57f767e5e46a458fc5d3c628d0cbb, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  GlobalObjectIdHash: 2963833585
+  InScenePlacedSourceGlobalObjectIdHash: 2294311777
+  DeferredDespawnTick: 0
+  Ownership: 1
+  AlwaysReplicateAsRoot: 0
+  SynchronizeTransform: 1
+  ActiveSceneSynchronization: 0
+  SceneMigrationSynchronization: 1
+  SpawnWithObservers: 1
+  DontDestroyWithOwner: 0
+  AutoObjectParentSync: 1
+  SyncOwnerTransformWhenParented: 1
+  AllowOwnerToParent: 0
 --- !u!4 &4274650840722361678
 Transform:
   m_ObjectHideFlags: 0
@@ -146753,6 +146899,22 @@
   m_hasFontAssetChanged: 0
   m_baseMaterial: {fileID: 0}
   m_maskOffset: {x: 0, y: 0, z: 0, w: 0}
+--- !u!1 &5277496091955932557
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 8757557227553398540}
+  m_Layer: 0
+  m_Name: Sectors
+  m_TagString: Untagged
+  m_Icon: {fileID: 0}
+  m_NavMeshLayer: 0
+  m_StaticEditorFlags: 0
+  m_IsActive: 1
 --- !u!4 &5318574659444547964
 Transform:
   m_ObjectHideFlags: 0
@@ -146797,6 +146959,31 @@
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 2147483647
   m_IsActive: 1
+--- !u!114 &5394173209623858289
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 7174552832594421700}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d5a57f767e5e46a458fc5d3c628d0cbb, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  GlobalObjectIdHash: 3860111171
+  InScenePlacedSourceGlobalObjectIdHash: 2294311777
+  DeferredDespawnTick: 0
+  Ownership: 1
+  AlwaysReplicateAsRoot: 0
+  SynchronizeTransform: 1
+  ActiveSceneSynchronization: 0
+  SceneMigrationSynchronization: 1
+  SpawnWithObservers: 1
+  DontDestroyWithOwner: 0
+  AutoObjectParentSync: 1
+  SyncOwnerTransformWhenParented: 1
+  AllowOwnerToParent: 0
 --- !u!1 &5397572915332759370
 GameObject:
   m_ObjectHideFlags: 0
@@ -146982,6 +147169,21 @@
   m_CorrespondingSourceObject: {fileID: -8524424544509167813, guid: 4d24d7ae160abb54984c80132db48961, type: 3}
   m_PrefabInstance: {fileID: 5479298408838670341}
   m_PrefabAsset: {fileID: 0}
+--- !u!4 &5503911054193018074
+Transform:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 7174552832594421700}
+  serializedVersion: 2
+  m_LocalRotation: {x: -0, y: -0.4071922, z: -0, w: 0.91334254}
+  m_LocalPosition: {x: -198.2, y: -11.483154, z: -37.44}
+  m_LocalScale: {x: 0.1, y: 5, z: 20}
+  m_ConstrainProportionsScale: 0
+  m_Children: []
+  m_Father: {fileID: 8757557227553398540}
+  m_LocalEulerAnglesHint: {x: 0, y: -48.057, z: 0}
 --- !u!1 &5525827730060895891
 GameObject:
   m_ObjectHideFlags: 0
@@ -147000,6 +147202,21 @@
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
   m_IsActive: 1
+--- !u!4 &5538467066630848006
+Transform:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 9095071193264868738}
+  serializedVersion: 2
+  m_LocalRotation: {x: -0, y: -0.09874328, z: -0, w: 0.99511296}
+  m_LocalPosition: {x: -4.388977, y: -11.483154, z: -0.14969635}
+  m_LocalScale: {x: 0.1, y: 5, z: 11}
+  m_ConstrainProportionsScale: 0
+  m_Children: []
+  m_Father: {fileID: 8757557227553398540}
+  m_LocalEulerAnglesHint: {x: 0, y: -11.334, z: 0}
 --- !u!20 &5589228197061130296
 Camera:
   m_ObjectHideFlags: 0
@@ -147910,6 +148127,21 @@
   m_Sources:
   - sourceTransform: {fileID: 1181191709}
     weight: 1
+--- !u!4 &6391786284930022340
+Transform:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 6919946283301097841}
+  serializedVersion: 2
+  m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
+  m_LocalPosition: {x: 267.93997, y: 27.07493, z: -103.8203}
+  m_LocalScale: {x: 1, y: 1, z: 1}
+  m_ConstrainProportionsScale: 0
+  m_Children: []
+  m_Father: {fileID: 0}
+  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
 --- !u!4 &6451863694048289873
 Transform:
   m_ObjectHideFlags: 0
@@ -148174,6 +148406,22 @@
   m_Children: []
   m_Father: {fileID: 8288199812752346961}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+--- !u!114 &6811973533881015892
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 7174552832594421700}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: e4cec0a24e334589955225595d5effbb, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  ShowTopMostFoldoutHeaderGroup: 1
+  isFinish: 1
+  leaderBoard: {fileID: 0}
+  sectorId: 2
 --- !u!114 &6812543674401136802
 MonoBehaviour:
   m_ObjectHideFlags: 0
@@ -148298,6 +148546,25 @@
   m_Children: []
   m_Father: {fileID: 8288199812752346961}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+--- !u!1 &6919946283301097841
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 6391786284930022340}
+  - component: {fileID: 703439785121964108}
+  - component: {fileID: 8361708472769000014}
+  - component: {fileID: 3407610695652645223}
+  m_Layer: 0
+  m_Name: LandstedeRacingManager
+  m_TagString: Untagged
+  m_Icon: {fileID: 0}
+  m_NavMeshLayer: 0
+  m_StaticEditorFlags: 0
+  m_IsActive: 1
 --- !u!224 &6941534878383001230
 RectTransform:
   m_ObjectHideFlags: 0
@@ -148578,6 +148845,25 @@
   m_FillOrigin: 0
   m_UseSpriteMesh: 0
   m_PixelsPerUnitMultiplier: 1
+--- !u!1 &7174552832594421700
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 5503911054193018074}
+  - component: {fileID: 5394173209623858289}
+  - component: {fileID: 6811973533881015892}
+  - component: {fileID: 1958306796285454634}
+  m_Layer: 0
+  m_Name: Sector
+  m_TagString: Untagged
+  m_Icon: {fileID: 0}
+  m_NavMeshLayer: 0
+  m_StaticEditorFlags: 0
+  m_IsActive: 1
 --- !u!4 &7192191282247670244
 Transform:
   m_ObjectHideFlags: 0
@@ -149497,6 +149783,27 @@
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 2147483647
   m_IsActive: 1
+--- !u!65 &8130749608476151645
+BoxCollider:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 9095071193264868738}
+  m_Material: {fileID: 0}
+  m_IncludeLayers:
+    serializedVersion: 2
+    m_Bits: 0
+  m_ExcludeLayers:
+    serializedVersion: 2
+    m_Bits: 0
+  m_LayerOverridePriority: 0
+  m_IsTrigger: 1
+  m_ProvidesContacts: 0
+  m_Enabled: 1
+  serializedVersion: 3
+  m_Size: {x: 1, y: 1, z: 1}
+  m_Center: {x: 0, y: 0, z: 0}
 --- !u!65 &8139941557679882551
 BoxCollider:
   m_ObjectHideFlags: 0
@@ -149862,6 +150169,31 @@
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
   m_IsActive: 1
+--- !u!114 &8361708472769000014
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 6919946283301097841}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d5a57f767e5e46a458fc5d3c628d0cbb, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  GlobalObjectIdHash: 73914611
+  InScenePlacedSourceGlobalObjectIdHash: 2294311777
+  DeferredDespawnTick: 0
+  Ownership: 1
+  AlwaysReplicateAsRoot: 0
+  SynchronizeTransform: 1
+  ActiveSceneSynchronization: 0
+  SceneMigrationSynchronization: 1
+  SpawnWithObservers: 1
+  DontDestroyWithOwner: 0
+  AutoObjectParentSync: 1
+  SyncOwnerTransformWhenParented: 1
+  AllowOwnerToParent: 0
 --- !u!1 &8389588638761537805
 GameObject:
   m_ObjectHideFlags: 0
@@ -150291,6 +150623,23 @@
   m_Children: []
   m_Father: {fileID: 0}
   m_LocalEulerAnglesHint: {x: 90, y: 90, z: 101}
+--- !u!4 &8757557227553398540
+Transform:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 5277496091955932557}
+  serializedVersion: 2
+  m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
+  m_LocalPosition: {x: 268.78897, y: 39.343155, z: -104.3003}
+  m_LocalScale: {x: 1, y: 1, z: 1}
+  m_ConstrainProportionsScale: 0
+  m_Children:
+  - {fileID: 5538467066630848006}
+  - {fileID: 5503911054193018074}
+  m_Father: {fileID: 0}
+  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
 --- !u!4 &8762634002074040930
 Transform:
   m_ObjectHideFlags: 0
@@ -150542,6 +150891,25 @@
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 2147483647
   m_IsActive: 1
+--- !u!1 &9095071193264868738
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 5538467066630848006}
+  - component: {fileID: 4138672750598196639}
+  - component: {fileID: 3500047205986739812}
+  - component: {fileID: 8130749608476151645}
+  m_Layer: 0
+  m_Name: Finish
+  m_TagString: Untagged
+  m_Icon: {fileID: 0}
+  m_NavMeshLayer: 0
+  m_StaticEditorFlags: 0
+  m_IsActive: 1
 --- !u!1 &9102070665486022502
 GameObject:
   m_ObjectHideFlags: 0
@@ -150608,3 +150976,5 @@
   - {fileID: 8721273540158958122}
   - {fileID: 1661715079}
   - {fileID: 333746969}
+  - {fileID: 6391786284930022340}
+  - {fileID: 8757557227553398540}
